homeassistant/components/climate/nest.py

```diff
@@ -78,24 +77,33 @@ def current_temperature(self):
         return self.device.temperature
 
     @property
-    def operation(self):
+    def current_operation(self):
         """Return current operation ie. heat, cool, idle."""
-        if self.device.hvac_ac_state is True:
+        if self.device.hvac_ac_state:
             return STATE_COOL
-        elif self.device.hvac_heater_state is True:
+        elif self.device.hvac_heater_state:
             return STATE_HEAT
         else:
             return STATE_IDLE
 
     @property
+    def target_temperature(self):
+        """Return the temperature we try to reach."""
+        if self.device.mode != 'range':
```

Comment:
What if `self.is_away_mode_on` is `True` ? In that case you will return both `target_temperature` and `target_temperature_low`